pipelines/docker-build-oci-ta.yaml

```
@@ -43,6 +43,10 @@ spec:
     description: Path to the Dockerfile inside the context specified by parameter path-context
     name: dockerfile
     type: string
+  - default: sentry-auth
```

Since the `build-container` task only supports one additional secret, some projects will likely need to combine multiple secrets into one. Therefore, I think we should start introducing a generic param value now. Then projects that used `sentry-auth` can either move their secret data to the new generic name, or override the param value to `sentry-auth` from their pipelines.
```suggestion
  - default: build-container-additional-secret
```

---

That was my idea as well, I didn't wanted to break the apps already using `sentry-auth` so instead of using the generic value I used `sentry-auth`. Apps which are already using this would not be broken and we can point other apps to use any name for their secrets.

---

Considering `sentry-auth` is fairly new, I'd like to check how many pipelines would actually be broken and whether we could synchronize the merge in multiple repos, if the repos number is reasonable.

---

https://github.com/search?q=org%3ARedHatInsights+sentry-auth&type=code